### PR TITLE
Tweak form css

### PR DESCRIPTION
**Pull request by thayashi**
Tweak to look similar to v1 layout

### CODE REVIEW

File: css/sass/dialog.scss [hunk 1]
```diff
@@ -197,7 +213,11 @@ $object-field-background: #ebf1f5;
 }
 
 .gohan-form-object-children {
-  padding-left: 10px;
+  // padding-left: 10px;
```
Comment (sealionkat) [inline css/sass/dialog.scss]: IMHO this padding helps user to see hierarchy in object. Without it, user may not know at once which fields are together in one object/group :slightly_smiling_face:

Comment (thayashi) [reply css/sass/dialog.scss]: I got your point, I fixed it. Please confirm.

**Comment by thayashi** [comment]
@sealionkat I fixed regarding your comment, Please confirm.

**Comment by sealionkat** [comment]
LGTM

**Pull request merged**